ai-services/cmd/ai-services/cmd/bootstrap/validate.go

```diff
@@ -103,6 +103,9 @@ func RunValidateCmd(skip map[string]bool) error {
 	for _, rule := range validators.DefaultRegistry.Rules() {
 		ruleName := rule.Name()
 		if skip[ruleName] {
+			if ruleName == CheckNUMA {
+				logger.Warningln("Skipping NUMA check, the tool may behave differently if NUMA node is not set to 1")
```

Comment:
can rephrase to `ai-services tool may not perform upto best of its capability if the NUMA node is not aligned to 1`